data-raw/wales-critical-general-acute-beds.R

```diff
@@ -0,0 +1,64 @@
+# ---- Load ----
+library(statswalesr)
+library(jsonlite)
+library(httr)
+library(RCurl)
+library(dplyr)
+library(readr)
+library(tidyr)
+library(lubridate)
+library(janitor)
+
+
+# ---- Function to download data from Stats Wales website ----
+raw <- statswales_get_dataset("hlth0310")
+
+
+# Clean the data
+wales_beds <-
+  raw |>
+  tibble() |> 
```

Comment:
Replace with `as_tibble()`. From the docs: 'Use as_tibble() to turn an existing object into a tibble.'